cmd/fetch.go

```diff
@@ -97,8 +110,85 @@ For more information, please refer to "getistio list --help" command.
 
 	flags := cmd.Flags()
 	flags.SortFlags = false
-	flags.StringVarP(&flagVersion, "version", "", "", "Version of istioctl e.g. \"--version 1.7.4\"")
-	flags.StringVarP(&flagFlavor, "flavor", "", "", "Flavor of istioctl, e.g. \"--flavor tetrate\" or --flavor tetratefips\" or --flavor istio\"")
-	flags.IntVarP(&flagFlavorVersion, "flavor-version", "", -1, "Version of the flavor, e.g. \"--version 1\"")
+	flags.StringVarP(&flag.Name, "name", "", "", "Name of distribution, e.g. 1.9.0-istio-v0")
+	flags.StringVarP(&flag.Version, "version", "", "", "Version of istioctl e.g. \"--version 1.7.4\". When --name flag is set, this will not be used.")
+	flags.StringVarP(&flag.Flavor, "flavor", "", "", "Flavor of istioctl, e.g. \"--flavor tetrate\" or --flavor tetratefips\" or --flavor istio\". When --name flag is set, this will not be used.")
+	flags.Int64VarP(&flag.FlavorVersion, "flavor-version", "", -1, "Version of the flavor, e.g. \"--version 1\". When --name flag is set, this will not be used.")
 	return cmd
 }
+
+func fetchParams(flags *FetchFlags,
+	ms *api.Manifest) (*api.IstioDistribution, error) {
+	if len(flags.Name) != 0 {
+		d, err := api.IstioDistributionFromString(flags.Name)
+		if err != nil {
+			return nil, fmt.Errorf("cannot parse given name %s to istio distribution", flags.Name)
+		}
+		return d, nil
+	}
+	if flags.Flavor != api.IstioDistributionFlavorTetrate && flags.Flavor != api.IstioDistributionFlavorTetrateFIPS && flags.Flavor != api.IstioDistributionFlavorIstio {
```

Comment:
shall we have new line somewhere in the middle of this line since it looks too long